### PR TITLE
Develop new

### PR DESCRIPTION
**Pull request by Ilya-Krylov**
* removed useless run* scripts that were part of test system
* hid web ote since it is not supported
* hid tf models inside Misc folder and mark them as unsupported
* hid pt models (non-template) inside Misc folder and mark some of them as unsupported
* divided ote tests and models tests

### CODE REVIEW

File: models/text_spotting/model_templates/alphanumeric-text-spotting/text-spotting-0003/template.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ hyper_parameters:
   basic:
     batch_size: 4
     base_learning_rate: 0.01
-    epochs: 25
+    epochs: 24
```
Comment (evgeny-izutov) [inline models/text_spotting/model_templates/alphanumeric-text-spotting/text-spotting-0003/template.yaml:34]: the easy fix) :1st_place_medal:

**Review by evgeny-izutov — APPROVED** [review]
Previously reviewed too, so no objection.

**Review by LeonidBeynenson — CHANGES_REQUESTED** [review]
2 @Ilya-Krylov :
Ilya, please, remove the function `run_tests_by_pattern` from the files `ote/ote/tests/utils.py` and `models/action_recognition_2/tests/common/utils.py`.
"This is not required any more"
Most probably, the entire file `models/action_recognition_2/tests/common/utils.py` should be removed too.

**Comment by Ilya-Krylov** [comment]
@LeonidBeynenson https://github.com/openvinotoolkit/training_extensions/pull/463

**Pull request merged**